.github/workflows/pipeline_pr.yaml

```diff
@@ -18,4 +18,17 @@ jobs:
       packages-auth-user: ${{secrets.PACKAGES_AUTH_USER}}
       packages-auth-token: ${{secrets.PACKAGES_AUTH_TOKEN}}
       connect-auth-user: ${{secrets.CONNECT_AUTH_USER}}
-      connect-auth-token: ${{secrets.CONNECT_AUTH_TOKEN}}
\ No newline at end of file
+      connect-auth-token: ${{secrets.CONNECT_AUTH_TOKEN}}
+  notify-on-error:
```

Comment:
Since the job before this job calls the `build_and_test` workflow, we should only keep the error notification in this workflow or the other workflow. The configurations look good in both, but if we have both implemented, then we would probably get notified twice since there would be a failure in the `build_and_test` workflow which would also cause a failure in this workflow as well